### DISCUSSION TITLE
System Menu - MAC address observatrion

### DISCUSSION THREAD

**Discussion by AcBCDN** [Guilouz/Creality-Helper-Script-Wiki#437] [Issues]
On my K1 MAX I have the wifi and the ethernet connected. When I go to the system menu the **IP Address** that is reported is for the **ETHERNET** port. However, the MAC address is for the **wlan0**.  

I use the MAC address for DHCP reservation on my router. I was a bit confused when I was not getting the correct MAC address.  

Thanks,

**Comment by Guilouz** [comment]
MAC address is retrieved by system_config.json file located in /usr/data/creality/userdata/config with device_mac parameter

This is the MAC address registered when you connect your printer to network. What is the interest to connect ethernet and wlan ?

**Reply by AcBCDN** [reply]
Good to know. Thanks!

I connected to ethernet and wlan to migrate the printer(s) to another network.  I used the two connections to have printer access between the two networks while migrating.  Easier to migrated stuff and still be able to access the printer from either network. 

Kinda handy; at times.

**Reply by Hammerfest** [reply]
Similar, I live in an over-crowded complex, WiFi sometime craps out, but the K1 Max's Ethernet has been unreliable for me (board being replaced, probably bad), so having redundancy is good 

It did take some time to get 2 MAC addresses
